server/src/test/java/io/pravega/schemaregistry/server/rest/ConfigTest.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.schemaregistry.server.rest;
+
+import io.pravega.test.common.AssertExtensions;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class ConfigTest {
+    @Test
+    public void testConfig() {
```

Comment:
It'd be useful to break down this single test into three different tests as there are different behaviors being tested. The first set verifies the defaults. The second set verifies input validations and the last set  verifies a valid buildout.